File: java/Practice01/src/myClient/MyConnection.java
package myClient;

import java.io.Closeable;

public class MyConnection {
    
	public final static int RECONNECT_INTERVAL = 3000;
	
    public MyConnection(String[] uris) {
        throw new RuntimeException("Not implemented");
    }
    
    public void open() {
        throw new RuntimeException("Not implemented");
    }
    
    public void close() {
        throw new RuntimeException("Not implemented");
    }

    public Closeable subscribe(int queryId, MySubscriber subscriber)
    {
        throw new RuntimeException("Not implemented");
    }
    
    public synchronized void addConnectionListener(MyConnectionEventListener listener) {
        throw new RuntimeException("Not implemented");
    }
    
    public synchronized void removeConnectionListener(MyConnectionEventListener listener) {
        throw new RuntimeException("Not implemented");
    }
}

File: csharp/Practice01/src-orig/MyClient/MyConnection.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace MyWork
{
    public class MyConnection
    {
        public readonly static int ReconnectInterval = 3000;

        public MyConnection(string[] uris)
        {
            throw new NotImplementedException();
        }

        public void Open()
        {
            throw new NotImplementedException();
        }

        public void Close()
        {
            throw new NotImplementedException();
        }

        public IDisposable Subscribe(int queryId, IMySubscriber subscriber)
        {
            throw new NotImplementedException();
        }

        public event EventHandler Connected;

        public event EventHandler ConnectionFailed;

        public event EventHandler Disconnected;
    }
}
